apps/keyboard/js/layouts/uk.js

```diff
@@ -0,0 +1,99 @@
+Keyboards.uk = {
+  label: 'Ukrainian',
+  shortLabel: 'Uk',
+  menuLabel: 'Українська',
+  imEngine: 'latin',
+  types: ['text', 'url', 'email', 'password'],
```

Comment:
Please remove 'password' here, since this keyboard would not contain all latin scripts, which are required for password input.